api.py

```diff
@@ -36,6 +37,19 @@ async def validate_schema_request_body(payload=Body(None)):
 async def validate_schema_from_url(url=None):
```

Comment:
This is not working for me at the moment? Getting the 400 error when I try to validate a schema from the questionnaire schemas repo? Is this working for you?

Reply:
No it cannot, I have updated it so it does not have the github.com domain 👍